covidthailand.py

```diff
@@ -2851,7 +2943,7 @@ def save_plots(df):
         ax=ax,
         y=rearrange(cols, *FIRST_AREAS),
```

Comment:
might need to pass in FIRST_AREAS and legend into plot_area to convert these to use plot_area

Reply:
Let's just pass the columns directly. The legend would need to be passed as well.

Reply:
ok. sounds good. so ditch the prefix shortcut or still keep that?

Reply:
See conversation below where I suggested an updated `def plot_area` function. So ditch the prefix and use `cols_subset` instead.